test/server/lib/HostedStorageManager.ts

```diff
@@ -962,6 +963,97 @@ describe('HostedStorageManager', function() {
       });
     });
   }
+
+  describe('minio-without-redis', async () => {
+    const workerId = 'dw17';
+    let tmpDir: string;
+    let oldEnv: EnvironmentSnapshot;
+    let docWorkerMap: IDocWorkerMap;
+    let externalStorageCreate: ExternalStorageCreator;
+    let defaultParams: ConstructorParameters<typeof HostedStorageManager>;
+
+    before(async function() {
+      tmpDir = await createTmpDir();
+      oldEnv = new EnvironmentSnapshot();
+      // Disable Redis
+      delete process.env.REDIS_URL;
```

Comment:
There's also a `TEST_REDIS_URL` I see mentioned in code, might that need wiping too?

Reply:
That should be fine - TEST_REDIS_URL seems to be used explicitly by other tests to configure where to connect to.

All we care about here is making sure we get DummyDocWorker out of `getDocWorkerMap`, which removing `process.env.REDIS_URL` is sufficient to do. :slightly_smiling_face:

Reply:
I'm afraid it gets set in test/testrun.sh in SaaS tests :( but yes it would be ok to deal with that if tests arrive there and are failing.